### PR TITLE
updated performance eval screen

### PR DESCRIPTION
**Pull request by jatnerubia**
Ticket ID: [Evaluations List Screen updates](https://github.com/orgs/nerubia/projects/18/views/2?pane=issue&itemId=45219434)

### CODE REVIEW

File: src/features/evaluations/UserEvaluationAdministrationsTable.tsx [hunk 1]
```diff
@@ -28,40 +30,43 @@ export const UserEvaluationAdministrationsTable = () => {
...
-      </table>
+              <h2 className='text-primary-500 text-lg font-semibold'>
+                {evaluationAdministration.name}
+              </h2>
+            </Button>
+            <p>
+              Evaluation Period:{" "}
+              {convertToMonthDate(evaluationAdministration.eval_period_start_date)} -{" "}
+              {convertToFullDate(evaluationAdministration.eval_period_end_date)}
+            </p>
```
Comment (emmaacha) [inline src/features/evaluations/UserEvaluationAdministrationsTable.tsx:51]: Is it possible that the periods will be on different years? If so, only the end date year is displayed here 🤔

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/features/evaluations/UserEvaluationAdministrationsTable.tsx (hunk 1 above)
Comment (jatnerubia) [reply src/features/evaluations/UserEvaluationAdministrationsTable.tsx:51]: can you ask sheila about this

Comment (emmaacha) [reply src/features/evaluations/UserEvaluationAdministrationsTable.tsx:51]: I think it's the same with this one, but yeah will ask ate Shei muna
image

Comment (emmaacha) [reply src/features/evaluations/UserEvaluationAdministrationsTable.tsx:51]: Just checked with ate Shei, format for different year should be mmm dd, yyyy - mmm dd, yyyy kuys @jatnerubia

Comment (emmaacha) [reply src/features/evaluations/UserEvaluationAdministrationsTable.tsx:51]: I mean, the format I pasted here applies to this page as well

Comment (jatnerubia) [reply src/features/evaluations/UserEvaluationAdministrationsTable.tsx:51]: alright

**Comment by jatnerubia** [comment]
@emmaacha done ^^

File: src/features/evaluations/UserEvaluationAdministrationsHeader.tsx
Comment (Dev-Nino) [inline src/features/evaluations/UserEvaluationAdministrationsHeader.tsx:1]: The filename should be in kebab case format.

File: src/features/evaluations/UserEvaluationAdministrationsTable.tsx
Comment (Dev-Nino) [inline src/features/evaluations/UserEvaluationAdministrationsTable.tsx:1]: The filename should be in kebab case format.

File: src/utils/formatDate.ts
Comment (Dev-Nino) [inline src/utils/formatDate.ts:1]: The filename should be in kebab case format?

**Review by Dev-Nino — CHANGES_REQUESTED** [review]
@jatnerubia meron po ako comment regarding lang sa file naming.

**Comment by jatnerubia** [comment]
@Dev-Nino done ^^

**Review by Dev-Nino — APPROVED** [review]

**Review by emmaacha — APPROVED** [review]

**Pull request merged**